docusaurus.config.js

```diff
@@ -199,7 +199,8 @@ module.exports = {
           { from: "/providers/", to: "/semgrep-ci/overview/" },
           { from: "/semgrep-app/role-based-access-control", to: "/semgrep-app/user-management/" },
           { from: "/cli-usage/", to: "/cli-reference/" },
-          { from: "/writing-rules/data-flow", to: "/writing-rules/data-flow/data-flow-overview" }
+          { from: "/writing-rules/data-flow", to: "/writing-rules/data-flow/data-flow-overview" },
+          { from: "/writing-rules/data-flow/overview/", to: "/writing-rules/data-flow/data-flow-overview" }
```

Comment:
```suggestion
          { from: "/writing-rules/data-flow/overview/", to: "/writing-rules/data-flow/data-flow-overview/" }
```

Reply:
Thanks! I should stick to `/` at the end.